curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/671682cd6d7aa95f0078f35f.md

```diff
@@ -0,0 +1,425 @@
+---
+id: 671682cd6d7aa95f0078f35f
+title: Step 67
+challengeType: 0
+dashedName: step-67
+---
+
+# --description--
+
+It is time to work on the final issue. If there is a to-do task with a special character like a 
+quote inside of the name or description of the item, the application breaks. While it appears otherwise, the 
+correct item will appear missing.  
+
+In order to fix this, we need to create a function called `removeSpecialChars` that takes a string as input
+and removes all special characters. 
```

Comment:
```suggestion
It is time to work on the final issue. If there is a to-do task with a special character like a quote inside of the name or description of the item, the application breaks. While it appears otherwise, the correct item will appear missing.  

In order to fix this, we need to create a function called `removeSpecialChars` that takes a string as input and removes all special characters. 
```